main/modes/mode_flight.c

```diff
@@ -0,0 +1,1374 @@
+                    char namebuff[FLIGHT_HIGH_SCORE_NAME_LEN+1];    //Force pad of null.
+                    memcpy( namebuff, name, FLIGHT_HIGH_SCORE_NAME_LEN );
+                    namebuff[FLIGHT_HIGH_SCORE_NAME_LEN] = 0;
+                    snprintf( buffer, sizeof(buffer), "%4s %3d.%02d", namebuff, cs/100,cs%100 );
+                    drawText( flight->disp, &flight->radiostars, CNDRAW_WHITE, buffer, anyp?81:17, (line+1)*10+10 );
+                }
+            }
+            break;
+        }
+        case FLIGHT_HIGH_SCORE_ENTRY:
+        {
+            int place = flightTimeHighScorePlace( flight->wintime, flight->beans >= MAX_BEANS );
+            textEntryDraw();
+
+            char placeStr[32] = {0};
+            snprintf(placeStr, sizeof(placeStr), "%d%s %s", place + 1, EnglishNumberSuffix[place],
+                (flight->beans == MAX_BEANS)?"100%":"ANY%" );
+            drawText( flight->disp, &flight->ibm, CNDRAW_WHITE, placeStr, 65,2);
+            break;
+        }
+    }
+
+    flightUpdateLEDs( flight );
+
+}
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+int16_t tdCOS( uint8_t iv );
+void tdIdentity( int16_t * matrix );
+static int16_t tdSIN( uint8_t iv );
+void Perspective( int fovx, int aspect, int zNear, int zFar, int16_t * out );
+int LocalToScreenspace( const int16_t * coords_3v, int16_t * o1, int16_t * o2 );
+void SetupMatrix( void );
+void tdMultiply( int16_t * fin1, int16_t * fin2, int16_t * fout );
+void tdRotateEA( int16_t * f, int16_t x, int16_t y, int16_t z );
+void tdScale( int16_t * f, int16_t x, int16_t y, int16_t z );
+void td4Transform( int16_t * pin, int16_t * f, int16_t * pout );
+void tdTranslate( int16_t * f, int16_t x, int16_t y, int16_t z );
+void Draw3DSegment( display_t * disp, const int16_t * c1, const int16_t * c2 );
+uint16_t tdSQRT( uint32_t inval );
+int16_t tdDist( const int16_t * a, const int16_t * b );
+
+
+//From https://github.com/cnlohr/channel3/blob/master/user/3d.c
+
+static const uint8_t sintable[128] = { 0, 6, 12, 18, 25, 31, 37, 43, 49, 55, 62, 68, 74, 80, 86, 91, 97, 103, 109, 114, 120, 125, 131, 136, 141, 147, 152, 157, 162, 166, 171, 176, 180, 185, 189, 193, 197, 201, 205, 208, 212, 215, 219, 222, 225, 228, 230, 233, 236, 238, 240, 242, 244, 246, 247, 249, 250, 251, 252, 253, 254, 254, 255, 255, 255, 255, 255, 254, 254, 253, 252, 251, 250, 249, 247, 246, 244, 242, 240, 238, 236, 233, 230, 228, 225, 222, 219, 215, 212, 208, 205, 201, 197, 193, 189, 185, 180, 176, 171, 166, 162, 157, 152, 147, 141, 136, 131, 125, 120, 114, 109, 103, 97, 91, 86, 80, 74, 68, 62, 55, 49, 43, 37, 31, 25, 18, 12, 6, };
```

Comment:
Don't wanna use `getSin1024()`?

Reply:
They are kinda different.  I guess I could try to coerse it.

Reply:
Oh yeah, I use BAMs, not degrees.

Reply:
What's a BAM? It's fine if you wanna stick with this table, I just wanted to put the idea forth.